doc/foreign-code.rst

```diff
@@ -257,6 +257,11 @@ descriptions by referencing them as the module specified in optional
    - ``(headers (preamble <preamble>)`` adds directly the preamble. Variables
      can be used in ``<preamble>`` such as ``%{read: }``.
 
+- Since the Dune's ``ctypes`` features is still experimental, it could be useful to
```

Comment:
```suggestion
- Since the Dune's ``ctypes`` feature is still experimental, it could be useful to
```
... or if 'features' is plural, the verb should be 'are,' like:
Since the Dune's ``ctypes`` features are still experimental...